CHANGELOG.md

```diff
@@ -18,7 +18,23 @@ and this project does not adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1] - 2014-11-15
 ### Added
-- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1 . 
+- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1. 
+
+## [0.2] - 2017-01-25
```

Comment:
Furthermore, the stuff in unreleased needs to be moved to this new "pre-merge" released, they cannot be left in "Unreleased" forever.